### PR TITLE
Add advanced logging config.

### LINKED ISSUE #74: Use syslogd for logging

### PR DESCRIPTION
**Pull request by neumantm**
Allows the end user to configure #74.
Closes #74

### CODE REVIEW

File: total_tolles_ferleihsystem/config.py [hunk 1]
```diff
@@ -13,10 +13,67 @@ class Config(object):
...
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     WEBPACK_MANIFEST_PATH = './build/manifest.json'
-    LOG_PATH = '/tmp'
-    LOG_FORMAT = '%(asctime)s [%(levelname)s] [%(name)-16s] %(message)s <%(module)s, \
-                 %(funcName)s, %(lineno)s; %(pathname)s>'
-    AUTH_LOG_FORMAT = '%(asctime)s [%(levelname)s] %(message)s'
+    LOG_LEVEL = logging.INFO
+    LOGGING = {
+        'version': 1,
+        'formatters': {
```
Comment (mee4895) [inline total_tolles_ferleihsystem/config.py]: Missing Task Logger?

File: total_tolles_ferleihsystem/config.py [hunk 2]
```diff
@@ -13,10 +13,67 @@ class Config(object):
...
+            },
+            'auth': {
+                'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
+            },
+            'lending': {
+                'format': '%(asctime)s [%(levelname)s] %(message)s',
+            }
+        },
+        'handlers': {
+            'default': {
```
Comment (mee4895) [inline total_tolles_ferleihsystem/config.py]: Missing Task Logger?

File: total_tolles_ferleihsystem/config.py [hunk 3]
```diff
@@ -13,10 +13,67 @@ class Config(object):
...
+            },
+            'lending' : {
+                'class' : 'logging.handlers.RotatingFileHandler',
+                'formatter' : 'lending',
+                'filename': '/tmp/ttf-lending.log',
+                'maxBytes' : 104857600,
+                'backupCount': 100,
+            },
+        },
+        'loggers': {
```
Comment (mee4895) [inline total_tolles_ferleihsystem/config.py]: Missing Default Logger?

File: total_tolles_ferleihsystem/config.py (hunk 1 above)
Comment (neumantm) [reply total_tolles_ferleihsystem/config.py]: Task logging uses the default formatter

File: total_tolles_ferleihsystem/config.py (hunk 2 above)
Comment (neumantm) [reply total_tolles_ferleihsystem/config.py]: Task logging uses the default handler. (As before)

File: total_tolles_ferleihsystem/config.py (hunk 3 above)
Comment (neumantm) [reply total_tolles_ferleihsystem/config.py]: The root logger is the default logger

**Pull request merged**